### PR TITLE
Change the doc to indicate both serving and eventing are required

### LINKED ISSUE #1468: Can't install if Knative Eventing is not installed
**Describe the bug**
Since the controller now depends on Knative Eventing CRDs, it's possible the controller will crash unless Knative Eventing is also installed. I haven't confirmed this.

 **Expected behavior**
I can install and create a knative-gcp source pointing directly at a consumer without also having Knative Eventing installed.

 **Additional context**
I'm not sure if the expected behavior above is actually expected or a high priority.

### PR DESCRIPTION
**Pull request by cathyzhyi**
Fixes #

## Proposed Changes

-
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: docs/install/install-knative-gcp.md [hunk 1]
```diff
@@ -55,11 +55,10 @@ export KO_DOCKER_REPO='gcr.io/my-gcloud-project-id'
    > if you are using Google Container Registry or `docker login` if you are
    > using Docker Hub.
 
-1. Install [Knative](https://knative.dev/docs/install/). Preferably, set up both
+1. Install [Knative](https://knative.dev/docs/install/). Set up
    [Serving](https://knative.dev/docs/serving/) and
-   [Eventing](https://knative.dev/docs/eventing/). The latter is only required
```
Comment (capri-xiyue) [inline docs/install/install-knative-gcp.md:60]: Do we need eventing if we don't use Pub/Sub `Channel` or a `Broker` backed by a Pub/Sub
   `Channel`?

Comment (cathyzhyi) [reply docs/install/install-knative-gcp.md:60]: According to https://github.com/google/knative-gcp/issues/1468#issuecomment-665286067 and https://github.com/google/knative-gcp/issues/1468#issuecomment-662111600 we need to install eventing but we might want to break this dependency in the future.

Comment (grac3gao) [reply docs/install/install-knative-gcp.md:60]: `gcp-broker` uses Eventing's `broker` CRD, it also needs eventing to be installed.

**Pull request merged**